testing/btest/coverage/test-all-policy-cluster.test

```diff
@@ -41,6 +33,8 @@ redef Log::default_rotation_interval = 0sec;
 
 event zeek_init()
 	{
+	# Prevent connections to the manager defined in Cluster::nodes.
+	delete Cluster::nodes[Cluster::node]$manager;
```

Comment:
Why was this not done with the original `manager` node?

Reply:
The original cluster-layout encoded in the file didn't have the `$manager` field on any of the other nodes, so they didn't attempt to establish a connection. The template does have `$manager` set, but we still don't want the outgoing connection.